packages/gatsby-plugin-static-image/src/image-processing.ts

```diff
@@ -0,0 +1,52 @@
+import { Node, GatsbyCache, Reporter } from "gatsby"
+import { fluid as fluidSharp, fixed as fixedSharp } from "gatsby-plugin-sharp"
+import fs from "fs-extra"
+import path from "path"
+
+const fileCache = new Map<string, Node>()
+
+export async function writeImages({
```

Comment:
❓ I would have the same remark for testing. I would find it valuable, but I'm a fresh player here 😓

Reply:
Absolutely agree